pkgs/development/tools/go-task/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+let
+  owner =
+    "go-task";
+
+  repo =
+    "task";
+
+in
+buildGoPackage rec {
```

Comment:
I looked at the upstream project and they include `go.sum` and go.mod`, so `buildGoModule` is probably the better choice here. Allows you to remove `deps.nix` as well.

Reply:
I did originally try that, it was complaining about failing to build `subPackages`. I assumed that was something to do with the unconventional build tool. I'm not a go dev though, so I'm not sure

Reply:
I have changed the derivation to use `buildGoModule`. It's only a small change compared to what you had before.

Reply:
Ahh so it was supposed to be `subPackages = [ "cmd/task" ];`. Sorry I don't have much experience with golang, I probably should have chosen something more familiar for my first contribution